src/api/ccapi/voms_api.cc

```diff
@@ -106,6 +104,26 @@ std::string vomsdata::ErrorMessage(void)
   return errmessage;
 }
 
+static pthread_once_t initialized = PTHREAD_ONCE_INIT;
+static bool ssl_is_initialized = false;
+
+static void initialize()
+{
+  if (!ssl_is_initialized) {
+     SSL_library_init();
```

Comment:
While technically this is not needed now it would much less confusing if you kept
ssl_is_initialized = true
so someone doesn't have to have the aha moment it's because of pthread_once!

Also, now would be a good time to also support 1.1.0 as outlined in
https://wiki.openssl.org/index.php/Library_Initialization

Reply:
I confess I forgot it and I had myself the aha moment after the fact. I'll add the set to true.
For the support to 1.1.0, we have already a branch for that (`develop-2.1.x`), which will become the new `develop`.